src/app/components/select/select.js

```diff
@@ -5,6 +5,8 @@ import {description as template} from './select.html';
 const CONVERT_OPTIONS = Symbol();
 const CLOSE_DROPDOWNS = Symbol();
 const CONTROLLERS = Symbol();
+const STOP_SCROLLING = Symbol();
```

Comment:
This use of symbols to create static methods/members seems like it isn't buying you anything over just creating functions or variables outside the class (but inside the module). They're just private functions and variables.